### DISCUSSION TITLE
Order of underlay application

### DISCUSSION THREAD

**Discussion by chubean** [inkstitch/inkstitch#2180] [Ideas]
I don't know if this was asked, but I'm wondering if the order of the underlays could be customized, i.e., zig zag first then contour.

**Comment by lexelby** [comment]
I want to some day!  That's on my personal feature list.

**Reply by chubean** [reply]
The options, features and tutorials to date are amazing!!  Much appreciation for all your work!!  I will keep an eye out for it.  Inkstitch is such a great tool for us!!
